source/codegen/metadata/nifpga/config.py

```diff
@@ -27,7 +27,8 @@
         'uint16_t': 'uint32',
         'uint32_t': 'uint32',
         'uint64_t': 'uint64',  
-        'NiFpga_Bool': 'bool'     
+        'NiFpga_Bool': 'bool',
+        'NiFpga_FifoProperty': 'uint32',
```

Comment:
Should grpc_type be `FifoProperty` enum here?

Reply:
I believe this should be uint32 only because, in the proto, it will be represented as shown below. We need one parameter to be of the _FifoProperty_ type and the other to be _raw_. If we set _FifoProperty_ here, the _raw_ parameter would also be of the _FifoProperty_ type, which I think is incorrect.

oneof property_enum {
    FifoProperty property = 3;
    uint32 property_raw = 4;
  }